docs/Automation.md

```diff
@@ -1,183 +1,196 @@
+
+##### ForProcessId (static)
+Create the builder for the config for the specified process.
+
+##### Parameters
+
+The **ForProcessId** method accepts the following parameters:
 
 **Name** | **Type** | **Description**
 ---|---|---
-OutputFile | `string`  | The location where output from this session will be written. The file extension can (and should) be omitted. This parameter can be specified either at Start-AxeWindows or at Invoke-Snapshot.
-OutputFileFormat | `string` | The output file format to generate. This parameter can be specified either at Start-AxeWindows or at Invoke-Snapshot.
-TargetProcessId | `string` | The process id, expressed as a string, that will be scanned. This parameter can be specified either at Start-AxeWindows or at Invoke-Snapshot.
-ConfigFile | `string` | The full path to a file that contains JSON-serialized Tier 3 parameters. Must point to a valid JSON-serialized file if specified.
+processId | `int` | The process Id of the application to test. If the value is invalid, the automation session will throw an [`AxeWindowsAutomationException`](#error-handling).
 
 ##### Return object
 
-The **StartCommandObject** has the following properties:
+The **ForProcessId** method returns an instance of **Config.Builder**.
 
-**Name** | **Type** | **Description**
----|---|---
-Completed | `bool` | True if the command ran to completion. If true, then the Succeeded property indicates if the call succeeded.
-SummaryMessage | `string` | A human-readable message to summarize the result. This message can change at any time, so do not rely on parsing it in tools.
-Succeeded | `bool` | True if the Start command completed successfully.
+##### WithOutputFileFormat
 
-#### Snapshot Command
-The Snapshot command scans a single process for accessibility issues and saves
-the output file for later analysis. It can only be called while an automation
-session is open (i.e., between the Start and Stop commands):
+Specify the type(s) of output files you wish AxeWindows to create.
 
-##### Inputs
+The [`OutputFileFormat` enum](../src/Automation/enums/OutputFileFormat.cs), currently has the following possible values:
 
-###### .NET
+**Name** | **Value** | **Description**
+---|---|---
+None | `0` | Create no output files.
+A11yTest | `1` | Create output files which can be opened using [Accessibility Insights for Windows](https://accessibilityinsights.io/docs/en/windows/overview).
 
-The **SnapshotCommand.Execute** method accepts the following parameters:
+##### Parameters
+
+The **WithOutputFileFormat** method accepts the following parameters:
 
 **Name** | **Type** | **Description**
 ---|---|---
-primaryConfig | `Dictionary<string, string>` | The set of Tier 1 parameters to apply during this call. See [Defined Constants](#defined-constants) for specific parameters.
+format | `OutputFileFormat` | The type(s) of output files you wish AxeWindows to create. No output files will be created if this is left unspecified. The default value is `None`.
```

Comment:
I think Axe.Windows does not serialize an a11ytest file unless there were errors. We should reflect this in the docs as well